examples/websocket_bench.py

```diff
@@ -0,0 +1,70 @@
+import asyncio
+import logging
+from sanic import Sanic
+from sanic.response import file
+from sanic.log import error_logger
+import time
+error_logger.setLevel(logging.INFO)
+app = Sanic(__name__)
+
+@app.route('/')
+async def index(request):
+    return await file('websocket_bench.html')
```

Comment:
We should use `app.static` if we want this to be an example implementation.

Reply:
I didn't mean to commit `websocket_bench.py` and `websocket_bench.html`, they were for my testing. We can leave them if you want, or I will remove them.
Note, this code is based on the [original `examples/websocket.py` file, which uses the same `file()` response](https://github.com/sanic-org/sanic/blob/404c5f9f9eb54f06d1f30250bc2cc840c18a0eab/examples/websocket.py#L9). Probably both need to be updated to use `app.static()`.

Reply:
It is nice to have them, but I would rather have them in a different repo since they are not really intended to be a "how to use Sanic". Personally, I would remove.